content/tutorials/drift-detection/index.md

```diff
@@ -0,0 +1,198 @@
+2. Copy the repository URL provided by GitHub.
+3. Replace the origin with your new GitHub repo using the following commands:
+
+```bash
+git init
+git add .
+git commit -m "Initial commit for my Pulumi Drift tutorial"
+git branch -M main
+git remote add origin https://github.com/your-username/learn-pulumi-drift-detection.git
+git push -u origin main
+```
+
+## Configure Drift Detection
+
+Now you will return to your browser to set your deployment configuration source control settings. Navigate to your stack `Settings` and `Deploy` tab. Next, configure the required Deployment settings, including your source control.
+
+1. Add your GitHub organization/repository and branch you created.
+
+2. Add your Secret environment variables, including `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
+
+{{% notes type="info" %}}
+Secret environment variables, such as `AWS_SECRET_ACCESS_KEY`, are encrypted end-to-end with Pulumi and can be set on each stack. However, by creating an [environment](/docs/pulumi-cloud/esc) with Pulumi ESC, you can centralize secrets and set up OIDC for secure authentication. This allows you to manage and share sensitive configuration data across multiple stacks efficiently.
+{{% /notes %}}
+
+Click **Save deployment configuration**.
+
+![Animation of how to configure drift detection in Pulumi](./pulumi-configure-drift.mp4)
+
+Once you have configured Pulumi Deployments, open the Actions menu and choose `Detect Drift`, then click `Deploy` to validate your stack, program and deployments are all running before we introduce drift.
+
+![Screenshot of the Pulumi Cloud console to run a Drift Detection](./pulumi-detect-drift-action.png)
+
+Once the drift run is complete, you should see a new Drift check on the stack `Drift` Tab
+
+## Introduce drift
+
+To see Drift Detection in action, you will now introduce drift by manually making changes to your newly created EC2 resource, in this case by adding a new tag.
+
+Log in to the AWS Management console with your browser and ensure you are in the correct AWS region where your Pulumi stack is deployed.
+
+Modify your EC2 Instance by going to the EC2 Dashboard, then select the EC2 instance named `webserver` and click on the **Tags** tab. Next, add a tag by clicking the **Manage tags** button. Then click the **Add new tag** button with the key `Description` and value of `Pulumi Drift Detection`. Finally, **Save** the changes.
+
+## Run a Drift Detection deployment
+
+Return to the Pulumi Cloud and your stack, and again choose the **Detect Drift** action and click **Deploy**.
+
+After your run completes you will see a warning icon on the Drift tab, indicating that drift has been detected. You can see a detailed summary of what resources have been updated or deleted, the properties that have changed, when the drift run happened and a link to the Deployment with further details.
+
+![Animation of the Pulumi Cloud console Drift tab](./pulumi-drift-detected.mp4)
```

Comment:
for some reason none of these mp4 files are rendering for me.

Reply:
fixed it; had to convert since I'm using mp4s now instead of animated gifs thx for the catch!